demo/lightning/image-segmentation/train.py

```diff
@@ -1,14 +1,12 @@
 import lightning as pl
 
-from model import Unet3D
+from model import Unet3DLightning
 from data import Unet3DDataModule
+from arguments import PARSER
 
 if __name__ == "__main__":
```

Comment:
Can you add a readme file to this demo explaining how to set up and run the demo and any additional information about it? It can come in a separate PR if that's easier.

Reply:
I will send out a separate PR that adds the instructions.